mautic/mautic
# TPROD-312 - CoreBundle Menu: migrate to twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
- Add & load Twig templates for menus as used by:
  - the left panel sidebars menus (and submenus)
  - the right panel sidebars menus (and submenus)
  - the user menu (top right, by the user avatar) [this one already existed]



#### Steps to test this PR:
- Check that the menus look correct and their links work
  - the left panel sidebars menus (and submenus):  
    - submenus should open/close
    - links go to correct pages
  - the right panel sidebars menus (and submenus):
    - links go to correct pages
  - the user menu (top right, by the user avatar):
    - links should go to correct pages
    - logout should work



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
